simulationdataschema/model_system_review.py

```diff
@@ -0,0 +1,865 @@
+        of the structure. It refers to a chemical element as defined in the periodic table,
+        e.g., 'H', 'O', 'Pt'. This quantity is equivalent to `atomic_numbers`.
+        """,
+    )
+
+    atomic_numbers = Quantity(
+        type=np.int32,
+        shape=["*"],
+        description="""
+        List of atomic numbers Z. This quantity is equivalent to `labels`.
+        """,
+    )
+
+    positions = Quantity(
+        type=np.float64,
+        shape=["*", 3],
+        unit="meter",
+        description="""
+        Positions of all the atoms in Cartesian coordinates.
+        """,
+    )
+
+    lattice_vectors = Quantity(
+        type=np.float64,
+        shape=[3, 3],
+        unit="meter",
+        description="""
+        Lattice vectors of the simulated cell in Cartesian coordinates. The first index runs
+        over each lattice vector. The second index runs over the $x, y, z$ Cartesian coordinates.
+        """,
+    )
+
+    lattice_vectors_reciprocal = Quantity(
+        type=np.float64,
+        shape=[3, 3],
+        unit="1/meter",
+        description="""
+        Reciprocal lattice vectors of the simulated cell, in Cartesian coordinates and
+        including the $2 pi$ pre-factor. The first index runs over each lattice vector. The
+        second index runs over the $x, y, z$ Cartesian coordinates.
+        """,
+    )
+
+    periodic_boundary_conditions = Quantity(
+        type=bool,
+        shape=[3],
+        description="""
+        If periodic boundary conditions are applied to each direction of the crystal axes.
+        """,
+    )
+
+    velocities = Quantity(
+        type=np.float64,
+        shape=["*", 3],
+        unit="meter / second",
+        description="""
+        Velocities of the atoms. It is the change in cartesian coordinates of the atom position
+        with time.
+        """,
+    )
+
+    supercell_matrix = Quantity(
+        type=np.int32,
+        shape=[3, 3],
+        description="""
+        Specifies the matrix that transforms the primitive unit cell into the supercell in
+        which the actual calculation is performed. In the easiest example, it is a diagonal
+        matrix whose elements multiply the `lattice_vectors`, e.g., [[3, 0, 0], [0, 3, 0], [0, 0, 3]]
+        is a $3 x 3 x 3$ superlattice.
+        """,
+    )
+
+    equivalent_atoms = Quantity(
+        type=np.int32,
+        shape=["*"],
+        description="""
+        List of equivalent atoms as defined in `labels`. If no equivalent atoms are found,
+        then the list is simply the index of each element, e.g.:
+            - [0, 1, 2, 3] all four atoms are non-equivalent.
+            - [0, 0, 0, 3] three equivalent atoms and one non-equivalent.
+        """,
+    )
+
+    wyckoff_letters = Quantity(
+        type=str,
+        shape=["*"],
+        # TODO improve description
+        description="""
+        Wyckoff letters associated with each atom position.
+        """,
```

Comment:
Just checked it out. That's some very strange approach. 2 points:

1. Since you are accessing `AtomicCell` anyhow, you can just generate Wyckoff on the fly for `Symmetry`. That is to say: the execution does not determine the section structure here. You can just shove the part of 1 normalizer to the next. Thematically, Wyckoff is Symmetry, period.
2. I'm even more convinced now that we should leave out `original` cell, as it isn't used in other normalizers. We should just extract it during parsing, then have functions that convert it to `primitive` or `conventional`, which we do store. We can discuss the actual implementation in-person.

Reply:
I think we discussed this already. In the previous schema, we had Symmetry.system_original, Symmetry.system_primitive and Symmetry.system_std.

I decided to move it out and create a repeated list of AtomicCell instead, to avoid potential issues on where the representative system is pointing (some times it was Atoms[0], some other times it was Symmetry.system_std[0]).

Now, for 1, that's is the main reason I am generating new AtomicCell sections (outside of Symmetry); this is not strange at all (at least not wrt to the previous way). But still, we need the SymmetryAnalyzer for generating this extra cells, that's why the method is appearing under Symmetry.

In any case, I didn't delve into the details of the SymmetryAnalyzer, but I know it is generating 2 cells from the originally parsed one. But this is an optional extraction; it might be that the primitive and conventional cells cannot be extracted using that package... thus we need to keep `"original"` as a fallback. Otherwise yes, the original will be redundant.

Reply:
> I decided to move it out and create a repeated list of AtomicCell instead, to avoid potential issues on where the representative system is pointing (some times it was Atoms[0], some other times it was Symmetry.system_std[0]).

Yeah, I see that now (mentioned it also in another thread). For now, it's fine, but imho, we should just store the primitive (original in case of no symmetry) cell with denotations of the symmetry operations, and a supercell matrix if that applies to the original. Conventional cells or any other kind of supercell can be produced on the fly, that isn't so hard. That's what I want to look at with @lauri-codes 

> we need the SymmetryAnalyzer for generating this extra cells, that's why the method is appearing under Symmetry.

Sure, but Wyckoff can still go under Symmetry (where it belongs) and then be extracted via the `SymmetryAnalyzer` that you call regardless...